llvm/docs/DeveloperPolicy.rst

```diff
@@ -424,6 +438,40 @@ squashing and merging PRs.
 For minor violations of these recommendations, the community normally favors
 reminding the contributor of this policy over reverting.
 
+Post-commit responsibilities
+----------------------------
+
+After landing a change, the committer is responsible for addressing any problems
+found in the future that the change is responsible for.  For example:
+
+* The code should compile cleanly on all supported platforms.
+
+* The changes should not cause any correctness regressions in the
+  `llvm-test-suite <https://github.com/llvm/llvm-test-suite>`_
+  and must not cause any major performance regressions.
+
+* The change set should not cause performance or correctness regressions for the
+  LLVM tools.
+
+* The changes should not cause performance or correctness regressions in code
+  compiled by LLVM on all applicable targets.
+
+* You are expected to address any `GitHub Issues
+  <https://github.com/llvm/llvm-project/issues>`_ that result from your change.
+
+We prefer for this to be handled before submission but understand that it isn't
+possible to test all of this for every submission.  Our build bots and nightly
+testing infrastructure normally finds these problems.  A good rule of thumb is
+to check the nightly testers for regressions the day after your change.  Build
```

Comment:
What nightly testers?

Reply:
This is stale text, but back in the day, llvm-test-suite was run nightly, I believe, and results were accumulated in LNT. [Docs](https://llvm.org/docs/lnt/intro.html) make reference to "nightly" there. I'll link it to make it clearer, but surely this could use rewriting.